cmd/goal/node.go

```diff
@@ -119,6 +120,26 @@ var nodeCmd = &cobra.Command{
 	},
 }
 
+func getMissingCatchPointLabel(dataDir string) []string {
+	client := ensureAlgodClient(dataDir)
+	vers, err := client.AlgodVersions()
+	if err != nil {
+		reportErrorf(errorNodeStatus, err)
+	}
+	genesis := strings.Split(vers.GenesisID, "-")
+	resp, err := http.Get("https://algorand-catchpoints.s3.us-east-2.amazonaws.com/channel/" + genesis[0] + "/latest.catchpoint")
```

Comment:
If you make this URL a constant and/or pass it into the function that will make testing easier because you can replace the URL with a test server.